frontend/src/metabase/selectors/user.ts

```diff
@@ -21,10 +23,10 @@ export const canManageSubscriptions = createSelector(
 
 export const getUserAttributes = createSelector(
   [getUser],
-  user => (user && user.login_attributes) || [],
+  user => user?.login_attributes || [],
 );
 
 export const getUserPersonalCollectionId = createSelector(
   [getUser],
-  user => (user && user.personal_collection_id) || null,
+  user => user?.personal_collection_id || null,
```

Comment:
With correct types I believe that `|| null` is not needed. `state.currentUser` can be `User | null`, so `state.currentUser?.personal_collection_id` will return either `null` or `personal_collection_id` which is `number`.

Reply:
But I understand that it might be safer to keep this check